complaint/templates/index.html

```diff
@@ -1,191 +1,209 @@
-                    <!-- Modal-->
-                    <div id="myModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true" class="modal fade text-left">
-                      <div role="document" class="modal-dialog">
-                        <div class="modal-content">
-                          <div class="modal-header">
-                            <h4 id="exampleModalLabel" class="modal-title">Fill Out a Complaint Form</h4>
-                            <button type="button" data-dismiss="modal" aria-label="Close" class="close"><span aria-hidden="true">×</span></button>
+      <ul class="sidebar-menu list-unstyled">
+        <li class="sidebar-list-item">
+          <a href="#" class="sidebar-link text-black"><span>HOME</span>
+          </a>
+        </li>
+      </ul>
+    </div>
+    <div class="page-holder w-100 d-flex flex-wrap">
+      <div class="container-fluid px-xl-5">
+        <section class="py-5">
+          <div class="row">
+
+            <!-- Form Elements -->
+            <div class="col-lg-12 mb-5">
+              <div class="card">
+                <div class="card-header text-center">
+                  <h3 class="h6 text-uppercase mb-0">COMPLAINTS MANAGEMENT MICROSERVICE TESTING</h3>
+                </div>
+                <div class="card-body">
+                  <h5 class="h6 mb-0">
+                    {% block content %}
+                    {% for complaint in value['result'] %}
+                    <article class="post">
+                      <header>
+                        {% if complaint.status == "new" %}
+                        <div class="h6 badge badge-success">
+                          {{ complaint.status }}
+                        </div>
+                        {% elif complaint.status == "closed" %}
+                        <div class="h4 badge badge-danger">
+                          {{ complaint.status }}
+                        </div>
+                        {% elif complaint.status == "resolved" %}
+                        <div class="h6 badge badge-primary">
+                          {{ complaint.status  }}
+                        </div>
+                        {% endif %}
+                        <!-- Delete icon -->
+                        <div class="text-right">
+                          <a href="{{ url_for('main_bp.delete', _id=complaint.id) }}">
+                            <ion-icon style="color:blue" ; name="trash-sharp"></ion-icon>
+
+                          </a>
+                        </div>
+                        <!-- Edit icon -->
+                        <div class="text-center">
+                          <button data-id="{{ complaint.id }}" data-select="form-button" type="button">
+                            <ion-icon size="large" name="create"></ion-icon>
+                          </button>
+                        </div>
+                        <form data-id="{{ complaint.id }}" data-select="form"
```

Comment:
The only changes i have done in this file is added `data-id` and `data-select` attributes in form and button element 

otherwise, everything else is due to formatter

Reply:
No problem